Src/Common/Domain/service/null/NullOrganizationUserService.cs

```diff
@@ -19,24 +19,18 @@ public virtual void AddOrganizationUser(IOrganizationUser organizationUser)
         {
             // do nothing
         }
+
         public virtual IOrganizationUser GetOrganizationUser(int recordId)
         {
             return null;
-
-        }
-        public virtual IOrganizationUser GetOrganizationUser(int organizationId, int userId)
-        {
-            return null; 
-        }
-        public virtual int GetOrganizationUserRole(int organizationId, int userId)
-        {
-            return -1; 
         }
+
         public virtual IOrganizationUser[] GetOrganizationUsers()
```

Comment:
Even if it's not a strict requirement we've defined, it seems helpful for allowing admins to assign roles to other organization users. I vote we leave it in.